handler/handler_test.go

```diff
@@ -1,56 +1,107 @@
 package handler_test
 
 import (
+	"bytes"
 	"context"
+	"io/ioutil"
+	"net/http"
 	"testing"
 
+	"github.com/ONSdigital/dp-elasticsearch/v2/awsauth"
+	"github.com/ONSdigital/dp-search-data-importer/config"
 	"github.com/ONSdigital/dp-search-data-importer/handler"
 	"github.com/ONSdigital/dp-search-data-importer/models"
+
+	dpelasticsearch "github.com/ONSdigital/dp-elasticsearch/v2/elasticsearch"
+	dphttp "github.com/ONSdigital/dp-net/http"
+	
 	. "github.com/smartystreets/goconvey/convey"
 )
 
-var expectedEvent1 = &models.SearchDataImportModel{
-	DataType:        "testDataType",
-	JobID:           "",
-	SearchIndex:     "ONS",
-	CDID:            "",
-	DatasetID:       "",
-	Keywords:        []string{"testkeyword1"},
-	MetaDescription: "",
-	Summary:         "",
-	ReleaseDate:     "",
-	Title:           "testTitle1",
-	TraceID:         "testTraceID",
-}
+var (
+	testContext = context.Background()
+
+	expectedEvent1 = &models.SearchDataImportModel{
+		DataType:        "testDataType1",
+		JobID:           "",
+		SearchIndex:     "ONS",
+		CDID:            "",
+		DatasetID:       "",
+		Keywords:        []string{"testkeyword1"},
+		MetaDescription: "",
+		Summary:         "",
+		ReleaseDate:     "",
+		Title:           "testTitle1",
+		TraceID:         "testTraceID1",
+	}
+
+	expectedEvent2 = &models.SearchDataImportModel{
+		DataType:        "testDataType2",
+		JobID:           "",
+		SearchIndex:     "ONS",
+		CDID:            "",
+		DatasetID:       "",
+		Keywords:        []string{"testkeyword2"},
+		MetaDescription: "",
+		Summary:         "",
+		ReleaseDate:     "",
+		Title:           "testTitle2",
+		TraceID:         "testTraceID2",
+	}
+
+	testEvents = []*models.SearchDataImportModel{
+		expectedEvent1,
+		expectedEvent2,
+	}
+
+	testSigner *awsauth.Signer
+
+	testCfg, _ = config.Get()
```

Comment:
You should handle the error otherwise the code could panick.

To be able to handle the error returned from config.Get() you must put it in a function and not a global variable.

Reply:
* @TODO added for error handling, as mentioned in previous comments
* removed use of config in handler